Gruntfile.js

```diff
@@ -368,7 +368,6 @@ module.exports = function(grunt) {
             'admin/**/*.js',
           ];
           const ignore = [
-            'ddocs/**/*.js',
```

Comment:
Add it now to `paths` array so that it can be linted as well. Currently it is not being linted.

Also if you would like to reduce the huge _Gruntfile_ by approx 8 lines you could create a `.eslintignore` and paste the contents of `const ignore = []`

Reply:
Whoops! That's embarrassing!

I think for now I'm OK with the ignore stuff being here, because it's so-situated with the white list.

I imagine that might change if either it became to unwieldy due to lots of exceptions, or we moved to a simpler code layout where the eslint whitelist was very short with specific exceptions dotted around the place